### PR TITLE
fixing captions page

### PR DESCRIPTION
**Pull request by caijim**
captions page icons looked ugly. they are slightly less ugly now

@caesar2164 @stvstnfrd 

screen shot 2015-06-02 at 5 50 17 pm

### CODE REVIEW

File: cms/static/sass/views/_captions.scss [hunk 1]
```diff
@@ -91,11 +93,7 @@
         .red {
           color: $red;
         }
-
-        .yellow {
-          background: $yellow;
-        }
-
+
```
Comment (caesar2164) [inline cms/static/sass/views/_captions.scss]: you have extra spaces here, remove please...

**Comment by stvstnfrd** [comment]
+1 for slightly less ugly pages!

**Comment by stvstnfrd** [comment]
Oh, and should I just cherry-pick the top commit? It looks like the first one on there is the one I already cherry-picked?

File: lms/templates/ccx/enrollment.html [hunk 2]
```diff
@@ -65,7 +65,7 @@
             <tr>
               <td>${member.student}</td>
               <td>${member.student.email}</td>
-              <td><div class="revoke"><i class="icon-remove-sign"></i> Revoke access</div></td>
+              <td><div class="revoke"><i class="fa-times-circle"></i> Revoke access</div></td>
```
Comment (caesar2164) [inline lms/templates/ccx/enrollment.html]: Actually, remove this from this PR.

File: lms/templates/ccx/grading_policy.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 <div id="warn-coach" class="wrapper-msg urgency-high warning">
   <div class="msg">
-    <i class="msg-icon icon-warning-sign"></i>
+    <i class="msg-icon fa-exclamation-triangle"></i>
```
Comment (caesar2164) [inline lms/templates/ccx/grading_policy.html]: Actually, remove this from this PR.

**Comment by caesar2164** [comment]
This is almost there, there's just a few niggles, as seen above...

File: lms/templates/ccx/enrollment.html (hunk 2 above)
Comment (stvstnfrd) [inline lms/templates/ccx/enrollment.html]: Good catch.

**Comment by caesar2164** [comment]
:+1:

**Comment by caesar2164** [comment]
@stvstnfrd I'm closing this in name of cleaning up our open PRs, the change is cherry-picked...

**Pull request closed without merging**